streaming/java/test.sh

```diff
@@ -7,33 +7,37 @@ set -x
 
 ROOT_DIR=$(cd "$(dirname "${BASH_SOURCE:-$0}")"; pwd)
 
-run_testng() {
-    "$@" || exit_code=$?
-    # exit_code == 2 means there are skipped tests.
-    if [ $exit_code -ne 2 ] && [ $exit_code -ne 0 ] ; then
-        exit $exit_code
-    fi
-}
-
 echo "build ray streaming"
 bazel build //streaming/java:all
 
+# Check that ray libstreaming_java doesn't include symbols from ray by accident.
+# Otherwise the symbols may conflict.
+symbols_conflict=$(nm bazel-bin/streaming/libstreaming_java.so | grep TaskFinisherInterface || true)
+if [[ -n "${symbols_conflict}" ]]; then
```

Comment:
Not a big deal but it'd be nice to use `[ ]` rather than `[[ ]]` since it's portable. (And it's also used elsewhere in the file.)

Reply:
fixed